study.md

```diff
@@ -228,7 +230,10 @@ List some criteria for well-designed code. Contrast this list with indicators
 of poor design.
 
 ```md
-<!-- your answer here -->
+You want your code to be DRY/concise as opposed to lengthy and repetitive...
+You also want your code to have one responsibility as opposed to doing too many things and being too vulnerable to change...
```

Comment:
Great way to phrase it — isolating as little responsibility to modules as possible.